components/homme/src/theta-l/share/prim_advance_mod.F90

```diff
@@ -1755,6 +1752,7 @@ subroutine compute_andor_apply_rhs(np1,nm1,n0,qn0,dt2,elem,hvcoord,hybrid,&
         enddo
 
         ! check for layer spacing <= 1m
+        if (scale3 /= 0) then
```

Comment:
When is scale3 zero? I looked at a couple of types, it is always 1?

Reply:
its used by sundials/arkode - it's the interface where compute_andor_apply_rhs() returns tendencies, instead of update state.